File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/ClaimInvoicePayment.cs
using RMA.Common.Entities;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;
using RMA.Service.ClaimCare.Contracts.Enums;

using System;

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class ClaimInvoicePayment : AuditDetails
    {
        public int ClaimId { get; set; }
        public int ClaimInvoiceType { get; set; }
        public string BenefitName { get; set; }
        public string BenefitCode { get; set; }
        public Decimal EstimatedValue { get; set; }
        public CoverMemberTypeEnum CoverMemberType { get; set; }
        public Decimal InvoiceAmount { get; set; }
        public Decimal AuthorizedAmount { get; set; }

        public DateTime? InvoiceDate { get; set; }
        public ClaimInvoiceStatus ClaimInvoiceStatus { get; set; }
        public bool included { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/funeral/claim-invoice-payment.ts
import { ClaimInvoiceStatusEnum } from "projects/shared-models-lib/src/lib/enums/claim-invoice-status.enum";
import { CoverMemberTypeEnum } from "projects/shared-models-lib/src/lib/enums/cover-member-type-enum";

export interface ClaimInvoicePayment {
  claimId: number;
  personEventId: string;
  claimInvoiceType: string;
  benefitName: string;
  benefitCode: string;
  estimatedValue: number;
  coverMemberType: CoverMemberTypeEnum;
  invoiceAmount: number;
  authorizedAmount: number;
  invoiceDate: Date | null;
  claimInvoiceStatus: ClaimInvoiceStatusEnum;
  included: boolean;
}